vars/checkoutQaInternal.groovy

```diff
@@ -4,6 +4,6 @@ def call(Map params = [:]) {
     dir("scylla-qa-internal") {
         git(url: 'git@github.com:scylladb/scylla-qa-internal.git',
             credentialsId:'b8a774da-0e46-4c91-9f74-09caebaea261',
-            branch: 'master')
+            branch: '3customers_simulate_data_profile_rolling_upgrade')
```

Comment:
what's in this branch ?, i.e. that we still need to merge ?

Reply:
Yes, it is not merged yet. Needs review